lib/linters/zero_unit.js

```diff
@@ -14,6 +14,23 @@ module.exports = {
         var number;
         var value;
         var unit;
+        var excludeProperties = ['opacity', 'z-index', 'line-height'];
+        var property = node.first('property').first('ident').content;
+
+        if (config) {
+            if (config.excludeProperties && config.excludeProperties.length) {
+                excludeProperties.push(config.excludeProperties);
```

Comment:
we'd only want to use concat if the intention was to return a new array as the result. since excludeProperties is just a reference point, I didn't see a need for that, and figured it could be modified in place safely.